### PR TITLE
[budi] update python sdk quickstart

### PR DESCRIPTION
**Pull request by budi**
Update the sdk Quickstart:
- Remove ServingRequestType
- Change default store names
- Remove granularity `none`

### CODE REVIEW

File: sdk/python/examples/quickstart/Quickstart.ipynb [hunk 1]
```diff
@@ -124,13 +123,12 @@
     "# Initialise client\n",
     "fs = Client(core_url=FEAST_CORE_URL, verbose=True)\n",
     "\n",
-    "serving_ds=Datastore(id='REDIS1')\n",
-    "warehouse_ds=Datastore(id='NOOP')\n",
+    "serving_ds=Datastore(id='SERVING')\n",
```
Comment (pradithya) [inline sdk/python/examples/quickstart/Quickstart.ipynb]: You could remove it actually, it will use the default store

Comment (budi) [reply sdk/python/examples/quickstart/Quickstart.ipynb]: removed on 9b33b80

**Pull request merged**